pages/add/add.scss

```diff
@@ -1,187 +1,143 @@
 .add-page {
-  h1,
-  h2 {
-    color: $bikeshed-magenta;
-    margin-bottom: 0.5rem;
+  .posted-by {
+    margin-top: $form-field-vertical-margin / 2;
   }
 
-  h1 {
-    font-size: 2.25rem;
-  }
-
-  h2 {
-    font-size: 1.5rem;
-    font-weight: 400;
-    margin-top: $form-field-vertical-margin;
-  }
-}
-
-.checkboxGroup {
-  label {
-    @extend .form-check-label;
-
-    margin-left: 0;
-  }
+  fieldset.creators {
+    .row {
+      vertical-align: middle;
+      margin: 0 0 0.5rem;
 
-  input {
-    @extend .form-check-input;
+      @media screen and (min-width: $bp-md) {
```

Comment:
This feels a bit...weird. Is there a way to do this with bootstrap classes? Modifying the base grid classes has a tendency to go south, in my experience. Looks like you'd have to at least start with the `.multiple` element having `.row` as well, so you're not creating a new `.row` element with each creator.

Reply:
Hey @alanmoo I agree with you but I think having a followup PR for this would be better. This chunk of CSS code is for the `/add` page and it was not newly introduced in this PR (the changes you saw are from indentation changes but the rules are the same).

Reply:
Sounds good